contracts/PartyBid.sol

```diff
@@ -167,6 +182,15 @@ contract PartyBid is ReentrancyGuardUpgradeable, ERC721HolderUpgradeable {
         auctionId = _auctionId;
         name = _name;
         symbol = _symbol;
+        // validate that party host won't retain the total token supply
+        uint256 _remainingBasisPoints = 10000 - TOKEN_FEE_BASIS_POINTS;
```

Comment:
should this be `10_000 -`?

might be simpler just t enforce partyHostBasisPoints < `9000`, so we don't need to deal with any math -- and we know that we will never take more than 10%